File: Ex43_math_exercises.java

public class Ex43_math_exercises{
	public static void main(String[] args){
		
		double x = Math.atan(Math.sin(Math.toRadians(32))*Math.cos(Math.toRadians(32)));
		System.out.println(x);
		
		double y = Math.pow((Math.exp(1.67))*(Math.pow(1.65,(Math.log(8))/(Math.log(20)))),1.0/4);
		System.out.println(y);
		
		double z = (Math.log(Math.pow(21, Math.log(35))))/(Math.log(7));
		System.out.println(z);
		
		double t = Math.exp(Math.sqrt(Math.log(4)) + Math.sqrt(Math.log(5)));
		System.out.println(t);
		
	}
}

File: 4th week math char string/Ex43_math_exercises.c
#include <stdio.h>
#include <math.h>

int main(){
	
	double x, y, z, t;
	
	x = atan(sin(32*M_PI/180)*cos(32*M_PI/180));
	printf("%lf\n",x);
	
	y = pow(exp(1.67)*pow(1.65,((log(8))/(log(20)))),0.25);
	printf("%lf\n",y);
	
	z = log(pow(21,log(35)))/log(7);
	printf("%lf\n",z);
	
	t = exp(sqrt(log(4))+sqrt(log(5)));
	printf("%lf\n",t);
	
	return 0;
}
